### ISSUE TITLE
Casing in enumerations

### ISSUE THREAD

**Issue by thespooler** [mganss/XmlSchemaClassGenerator#166]
I'm not quite sure, but shouldn't enumerations be case sensitive?
For example:
```xml
<?xml version="1.0" encoding="utf-8"?>
<xs:schema id="test" targetNamespace="urn:test" 
    xmlns="urn:test" 
    xmlns:xs="http://www.w3.org/2001/XMLSchema">
    <xs:simpleType name="TestCasing">
        <xs:restriction base="xs:string">
            <xs:enumeration value="none" />
            <xs:enumeration value="None" />
        </xs:restriction>
    </xs:simpleType>
</xs:schema>
```
This should be valid, but it generates an invalid C# enum because of casing transformation.
```c#
    public enum TestCasing
    {
        [System.Xml.Serialization.XmlEnumAttribute("none")]
        None,
        
        None,
    }    
```

**Comment by mganss** [comment]
Does using `-a-` help (disable pascal case) help? Note that this option applies globally so you'll have lowercase property  names as well.

**Comment by mganss** [comment]
This got fixed in #190
